assets/css/payment_template.css

```diff
@@ -0,0 +1,297 @@
+	transform: scale(1.1);
+	z-index: 99;
+}
+
+
+.card__title {
+	color: #304ffe;
+	display: inline;
+	font-family: "Days One", sans-serif;
+	font-size: 20px;
+	font-weight: 700;
+	letter-spacing: 0;
+	line-height: 28px;
+}
+
+.card__logo {
+	float: right;
+	height: 23px;
+}
+
+.card__price {
+	align-items: center;
+	color: #a2acb2;
+	display: flex;
+	font-size: 12px;
+	justify-content: space-between;
+	letter-spacing: 0;
+	line-height: 17px;
+}
+
+
+.card__price .price {
+	color: #00112f;
+	font-size: 24px;
+	font-weight: 600;
+	letter-spacing: 0;
+	line-height: 33px;
+	margin: 20px 0;
+}
+
+.card__features {
+	align-items: center;
+	color: #23282d;
+	display: flex;
+	font-size: 8px;
+	justify-content: space-between;
+	letter-spacing: 0;
+	line-height: 11px;
+	margin: 10px 0;
+}
+
+.card__features:last-of-type {
+	margin-bottom: 30px;
+}
+
+.button-choose-plan {
+	background: transparent linear-gradient(265deg, #d500f9 0, #2962ff 99%) 0 0 no-repeat padding-box;
+	border-radius: 16px;
+	color: #fff;
+	cursor: pointer;
+	font-family: "Days One", sans-serif;
+	font-size: 12px;
+	letter-spacing: 0;
+	line-height: 12px;
+	margin-top: auto;
+	padding: 10px;
+	text-align: center;
+}
+
+.current-plan {
+	background: #90a4ae 0 0 no-repeat padding-box;
+	border-radius: 16px;
+	color: #fff;
+	font-family: "Days One", sans-serif;
+	font-size: 12px;
+	letter-spacing: 0;
+	line-height: 12px;
+	margin-top: auto;
+	padding: 10px;
+	text-align: center;
+}
+
+.disable-downgrade {
+	align-items: center;
+	color: #ff1744;
+	display: flex;
+	font-family: "Open Sans Semibold", sans-serif;
+	font-size: 10px;
+	height: 37px;
+	margin-top: auto;
+	text-align: left;
+}
+
+
+.features-list {
+	color: #f9f9f9;
+	list-style: none;
+	margin-left: 20px;
+	padding: 30px 5px;
+}
+
+.features-list li::before {
+	color: #ff0;
+	content: "\2022";
+	display: inline-block;
+	font-weight: 700;
+	margin-left: -1em;
+	width: 1em;
+}
+
+.features-list li {
+	color: #fff;
+	font-size: 12px;
+	letter-spacing: 0;
+	line-height: 17px;
+	margin: 5px 0;
+}
+
+.unsubscribe-section {
+	color: #fff;
+}
+
+.confirm-change-section {
+	width: 100%;
+}
+
+.unsubscribe-section .buttons,
+.confirm-change-section .buttons {
+	align-items: center;
+	display: flex;
+	justify-content: center;
+}
+
+.unsubscribe-section .buttons .button,
+.confirm-change-section .buttons .button {
+	background: #ff0;
+	border-radius: 16px;
+	color: #00112f;
+	cursor: pointer;
+	font-family: "Days One", sans-serif;
+	font-size: 12px;
+	letter-spacing: 0;
+	line-height: 12px;
+	margin: 10px;
+	padding: 10px;
+	text-align: center;
+	width: 20%;
+}
+
+.unsubscribe-section .buttons .unsubscribe,
+.confirm-change-section .buttons .unsubscribe {
+	background: none;
+	border: 2px #ff0;
+	border-style: solid;
+	color: #ff0;
+}
+
+#message {
+	color: #fff;
+	height: 30px;
+	text-align: center;
+}
+
+.card_plan {
+	display: block;
+	margin: 0 auto;
+	height: 72px;
+}
+
+.margin_medium {
+	margin-top: 68px;
+}
+
+.old_price {
```

Comment:
Codacy found an issue: [Selector should use lowercase and separate words with hyphens (selector-class-pattern)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5907666)